README.rst

```diff
@@ -232,16 +227,53 @@ Operations are long-running jobs that are not executed within the time frame
 of an API call. For such operations, a task is created, and the API can be used
 to poll the status of the task.
 
+Move files
+
 .. code:: python
 
+    >>> import logging
     >>> from smartfile import BasicClient
+    >>>
     >>> api = BasicClient()
-    >>> t = api.post('/path/oper/move/', src='/foobar.png', dst='/images/foobar.png')
+    >>>
+    >>> LOGGER = logging.getLogger(__name__)
+    >>> LOGGER.setLevel(logging.INFO)
+    >>>
+    >>> api.move('file.txt', '/newFolder')
+    >>>
     >>> while True:
-    >>>     s = api.get('/task', t['uuid'])
-    >>>     if s['status'] == 'SUCCESS':
+    >>>     try:
+    >>>         s = api.get('/task', api['uuid'])
+    >>>         # Sleep to assure the server doesn't get overloaded
```

Comment:
Minor verbiage suggestion; instead of insinuating that we can "overload" the server, it might be better to make it clear that the user can get rate limited from sending too many calls.